src/SBVRType.d.ts

```diff
@@ -0,0 +1,47 @@
+interface SBVRType<I,O> {
+  types: ConcreteTypes
+
+  validate(value: any, required: boolean, cb:Callback<I>): void
```

Comment:
Here and in many places: inconsistent spaces before the type (I'm for having it everywhere)